### DISCUSSION TITLE
Support for ST7701S and RM67162

### DISCUSSION THREAD

**Discussion by TopherTheVegan** [Bodmer/TFT_eSPI#2647] [New processor/display support or capability request]
Are these ST7701S and RM67162 drivers in the works? I saw another post for the ST7701 to get added in time. Id be willing to send the lilygo displays to test and develop drivers for them as I love this library and want to use it for my next 2 projects with these unsupported chips. Thank you for your time.

**Comment by Bodmer** [comment]
I hope to find the time to do this but I am busy with other priorities at the moment.

**Reply by TopherTheVegan** [reply]
Well, thanks for taking the time to respond!

**Comment by profporridge** [comment]
+1 for RM67162 support

**Reply by MikeTheGent** [reply]
+1 for RM67162 from me as well.

**Comment by il0vew33d** [comment]
+1 for RM67162 support

**Comment by WombatHollow** [comment]
I also vote for RM67162 support.  Been trying to set up a setup for the New Lillygo T-Display-S3-AMOLED board in PlatformIO on VSC Development Environment based on VolosR examples where he uses the Sprite functionality of TFT_eSPI to write in memory and then lcd_PushColors method from RM67162.ccp driver to transfer to display.
But a lot of issues with TFT_eSPI redefining values set by RM67162 at compile time.
Thanks

**Comment by Elcid8687** [comment]
+ for ST7701S support

**Reply by wallydeedo** [reply]
Please please! X

**Comment by qiqiji** [comment]
Please take a look into qspi support, screen such as st77903

there is a driver under esp-idf, i tested works fine ( it cames with lvgl port)
https://github.com/espressif/esp-dev-kits/files/12249442/ST77903_QSPI_1_6_400_400.zip

this one may be newer but i cant compiling
https://github.com/espressif/esp-iot-solution/tree/a3a1caf09bd4026930521bf9a5a9fd529e2e1cbd/components/display/lcd/esp_lcd_st77903

**Reply by lpzsn8888** [reply]
+1 for RM67162 SPI support

**Comment by davepl** [comment]
Would also love a TFT_eSPI solution for 67162!

**Comment by APF-MRoger** [comment]
+1 for ST7701 ! Plenty of nice Waveshare screens with that driver.

**Comment by Allenjwb** [comment]
+1 for ST7701S support.Please！Please！Please！
